windows/scripts/extract_kernels.py

```diff
@@ -53,33 +54,7 @@
 
 SEVEN_ZIP_VERSION_REGEX = re.compile(r'7-Zip .*?(?P<major>\d+)\.(?P<minor>\d+)')
 
-
-# Maps Windows ISOs (taken from the supported images listed in
-# s2e/guest-images/images.json) to the location of the kernel within the ISO.
-#
-# The kernel executable is typically stored within an intermediary container.
-# The path to this intermediary container is the mapped value in this
-# dictionary.
-KRNL_CONTAINER_MAP = {
-    'en_windows_xp_professional_with_service_pack_3_x86_cd_x14-80428.iso':
-        [os.path.join('I386', '*.EX_')],
-    'en_windows_7_enterprise_with_sp1_x64_dvd_u_677651.iso':
-        [os.path.join('sources', 'install.wim')],
-    'en_windows_8_1_enterprise_x64_dvd_2971902.iso':
-        [os.path.join('sources', 'install.wim')],
-    'en_windows_10_enterprise_version_1703_updated_march_2017_x64_dvd_10189290.iso':
-        [os.path.join('sources', 'install.wim')],
-}
-
-# We use the wildcard pattern "nt*.exe" to find kernel executables in a Windows
-# ISO. Unforuntaly, this can also match other executables that are not related
-# to the kernel. A blacklist of these executables is maintained here.
-NT_BLACKLIST = [
-    'ntprint.exe',
-    'ntbackup.exe',
-    'ntvdm.exe',
-]
-
+NT_PATTERN = re.compile('(ntoskrnl|ntkrnlmp|ntkrnlpa)$')
```

Comment:
Even though you don't have to escape any special character, it's still good practice to use `r'(ntoskrnl|ntkrnlmp|ntkrnlpa)$'` for regex strings

Reply:
Does it make more sense to call this constant `KERNEL_PATTERN` or something like that?